examples/sampling/graphbolt/rgcn/hetero_rgcn.py

```diff
@@ -133,17 +138,6 @@ def create_dataloader(
         node_feature_keys["institution"] = ["feat"]
     datapipe = datapipe.fetch_feature(features, node_feature_keys)
 
-    # Move the mini-batch to the appropriate device.
```

Comment:
so in the latest example, we only support
1. cpu sampling+ cpu training
2. gpu sampling + gpu training
The hack here is for cpu sampling + gpu training. If so, it's ok to remove the hack.

Reply:
Yeah, that seems to be the case.